### DISCUSSION TITLE
Differentiating incoming vs outgoing network requests in traces

### DISCUSSION THREAD

**Discussion by govindrai** [jaegertracing/jaeger#4047] [Q&A] [answered]
Here's a screenshot of a trace in Jaegar that was generated by running the official [OTel JS HTTP Instrumentation example](https://github.com/open-telemetry/opentelemetry-js/tree/main/examples/http)
![image](https://user-images.githubusercontent.com/13859249/201487035-90221f29-38f2-47e6-9d26-b60bf5d1a9bd.png)

What I lose in this trace is information on whether request is an incoming request or outgoing request. Would it make sense to change the operation name of span to be [INCOMING|OUTGOING] HTTP [HTTP.METHOD] based on spanKind? For example in this case: OUTGOING HTTP GET and INCOMING HTTP GET?

FYI, I asked this question on [Open Telemetry JS](https://github.com/open-telemetry/opentelemetry-js/issues/3378#issuecomment-1301710420) first and they pointed me here.

**Comment by yurishkuro** [comment, marked as answer]
You can reason about this based on the change in the service name between parent and child spans.
